src/Elastic.Apm/Config/AbstractConfigurationWithEnvFallbackReader.cs

```diff
@@ -175,7 +175,7 @@ public virtual IReadOnlyList<Uri> ServerUrls
 			ParseVerifyServerCert(Read(KeyNames.VerifyServerCert, EnvVarNames.VerifyServerCert));
 
 		public bool EnableOpenTelemetryBridge =>
```

Comment:
Renaming the property would be a breaking change to the public API, right?
While it would be nice for consistency's sake, it's probably not worth the hassle.

Reply:
Hmm - that's a good point, I have not thoughts about that part.

Yes, I agree with you, I'd keep it as is to minimize impact on users.